.github/workflows/sbom.yaml

```
@@ -19,7 +19,7 @@ jobs:
         run: |
           TAG=${{ github.event.inputs.tag }}
           echo "TAG=$TAG" >> "$GITHUB_ENV"
-      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v3
+      - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v3
```

I didnt update the comment here in the original commit:

```suggestion
      - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
```